src/config_web.py

```diff
@@ -11,14 +11,15 @@
 # *****************************************************************************
 # Elasticsearch Settings
 # *****************************************************************************
-ES_HOST = 'localhost:9200'
-ES_INDEX = 'mygene_current'
-ES_DOC_TYPE = 'gene'
+# ES_HOST = 'localhost:9200'
+ES_HOST = "es8.biothings.io:9200"
```

Comment:
@ctrl-schaff I believe we should revert it back to `localhost:9200` when committing changes.

Reply:
good catch thank you